src/snowflake/snowpark/udf.py

```diff
@@ -854,14 +855,42 @@ def _do_register_udf(
         _emit_ast: bool = True,
         **kwargs,
     ) -> UserDefinedFunction:
+        if kwargs.get("_registered_object_name") is not None:
+            ast, ast_id = None, None
+            if _emit_ast:
+                stmt = self._session._ast_batch.assign()
+                ast = with_src_position(stmt.expr.udf, stmt)
+                ast_id = stmt.var_id.bitfield1
+
+            return UserDefinedFunction(
+                func,
+                return_type,
+                input_types or [],
+                kwargs["_registered_object_name"],
+                _ast=ast,
+                _ast_id=ast_id,
+            )
 
         check_imports_type(imports, "udf-level")
 
-        # AST. Capture original parameters, before any pre-processing.
-        ast = None
+        # get the udf name, return and input types
```

Comment:
nit: `Get the UDF name, return and input types.`

Arthur is a fan of sentences that start with Uppercase letter and end with a dot.

Reply:
Sure, this is the existing comment but I can clean it up.